src/app/annotationViewer/positionLine.js

```diff
@@ -0,0 +1,117 @@
+angular
+    .module("bawApp.annotationViewer.positionLine", [])
+    .directive("positionLine", ["bawApp.unitConverter", "$window", function (unitConverter, $window) {
+
+        return {
+            restrict: "A",
+            templateUrl: "annotationViewer/positionLine.tpl.html",
+            replace: true,
+            scope: {
+                media: "<media",
+                audioData: "<audioData",
+                imageClass: "<imageClass"
+            },
+            link: function (scope, elements, attributes, controller) {
+
+                var image;
+
+                /**
+                 * Searches for the closest node to the position line element that
+                 * has the class scope.imageClass
+                 * @param element
+                 * @returns DOM element or NULL
+                 */
+                this.getImageElement = function (element, depth) {
+
+                    if (depth > 3 || element.tagName === "BODY") {
+                        return null;
+                    }
+
+                    var containingElement = element.parentElement;
+                    var images = containingElement.getElementsByClassName(scope.imageClass);
+                    if (images.length > 0) {
+                        return images[0];
+                    } else {
+                        return this.getImageElement(containingElement, depth + 1);
+                    }
+                };
+
+
+                /**
+                 * converts the proportion of elapsed audio to a pixel value,
+                 * based on the width of the spectrogram image.
+                 * @param audioPositionSeconds number
+                 * @returns number
+                 */
+                scope.secondsToPixels = function (audioPositionSeconds) {
```

Comment:
Again tag this for deprecation and explain. We want a single source of truth for conversions - the UnitConverter